src/presentation/http/router/note.test.ts

```diff
@@ -39,13 +99,15 @@ describe('Note API', () => {
         },
         tools: [
           {
-            name: 'header',
+            name: headerTool.name,
+            exportName: headerTool.exportName,
             source: {
```

Comment:
why not to use `headerTool` and `paragraphTool` here instead of manually specifying of `name`, `exportName`, `source`, etc.